src/main/scala/za/co/absa/commons/parsers/UrisConnectionStringParser.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2020 ABSA Group Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package za.co.absa.commons.parsers
+
+object UrisConnectionStringParser {
```

Comment:
What is the use-case for this utility? Can it be considered generic enough to belong commons?

Reply:
I decided to delete it as indeed it seems to be too Enceladus-specific and similar functionality can be achieved with less effort in other/future projects.